charts/ratify/README.md

```diff
@@ -55,6 +55,10 @@ $ helm upgrade -n gatekeeper-system [RELEASE_NAME] ratify/ratify
 | vulnerabilityreport.disallowedSeverities           | List of severities to disallow (strings). Common severities: `low`, `medium`, `high`, `critical`, `unknown`                                                                                                                                                                                                                                                            | `[]`                              |
 | vulnerabilityreport.denylistCVEs                   | List of CVE IDs that cannot exist in the vulnerability report                                                                                                                                                                                                                                                                                                          | `[]`                              |
+| sbom.enabled                        | Enables/disables installation of sbom verification configuration                                                                                                                                                                                                                                                                                                       | `false`                           |
+| sbom.notaryProjectSignatureRequired                        | requires validation of sbom notation signature                                                                                                                                                                                                                                                                                                         | `false`                           |
+| sbom.disallowedLicenses                        | list of disallowed licenses                                                                                                                                                                                                                                                                                                     | []                           |
+| sbom.disallowedPackages                       | list of disallowed packages defined by package name and version                                                                                                                                                                                                                                                                                                               | []                           |
```

Comment:
wonder if we should provide an example for package name/version pair?

Reply:
+1. You could consider pointing to the doc if the plan is to add that there

Reply:
thanks , I will merge this PR first to unblock release changes. and will add the example in a separate PR.